proc_dash/utility.py

```diff
@@ -22,6 +22,15 @@ def construct_legend_str(status_desc: dict) -> str:
     )
 
 
+def construct_summary_str(data: pd.DataFrame) -> str:
+    """Creates summary of key counts for dataset."""
+    dataset_summary = f"""Total number of participants: {count_unique_subjects(data)}
+Total number of unique records (participant-session pairs): {count_unique_records(data)}
+Total number of unique sessions: {data["session"].nunique()}"""
+
+    return dataset_summary
```

Comment:
very :cherries: now, but I think here you could really just return the f string and not assign the temp variable